wiki/Tournaments/Country_Rankings/en.md

```diff
@@ -0,0 +1,361 @@
+| ![][flag_PL] Poland | ![][true] | ![][true] | ![][true] | ![][true] (3rd) | ![][true] (2nd) | ![][true] (3rd) | ![][true] | ![][true] (1st) |
+| ![][flag_PT] Portugal | ![][true] | ![][true] | ![][true] | ![][false] | ![][true] | ![][true] | ![][false] | ![][false] |
+| ![][flag_RO] Romania | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][true] |
+| ![][flag_RU] Russian Federation | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
+| ![][flag_SG] Singapore | ![][false] | ![][false] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
+| ![][flag_KR] South Korea | ![][true] (3rd) | ![][true] (1st) | ![][true] (1st) | ![][true] (1st) | ![][true] | ![][true] | ![][true] (3rd) | ![][true] |
+| ![][flag_ES] Spain | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][true] | ![][false] |
+| ![][flag_SE] Sweden | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
+| ![][flag_TW] Taiwan | ![][true] (1st) | ![][true] | ![][true] (3rd) | ![][true] (2nd) | ![][true] | ![][true] | ![][true] | ![][true] |
+| ![][flag_TH] Thailand | ![][false] | ![][true] | ![][true] | ![][true] | ![][false] | ![][true] | ![][false] | ![][false] |
+| ![][flag_TR] Turkey | ![][false] | ![][false] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] |
+| ![][flag_UA] Ukraine | ![][true] | ![][true] | ![][true] | ![][false] | ![][true] | ![][true] | ![][true] | ![][false] |
+| ![][flag_GB] United Kingdom | ![][true] | ![][false] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] (3rd) |
+| ![][flag_US] United States | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] (1st) | ![][true] (1st) | ![][true] (2nd) |
+| ![][flag_UY] Uruguay | ![][false] | ![][true] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] |
+| ![][flag_VE] Venezuela | ![][false] | ![][false] | ![][false] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] |
+| ![][flag_VN] Vietnam | ![][false] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] |
+| Total number of teams | 28 | 32 | 32 | 32 | 32 | 32 | 32 | 32 |
+
+### ![osu!taiko icon](/wiki/shared/mode/taiko.png) osu!taiko
+
+#### Ranking
+
+| Rank | Country | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
+| :-: | --- | :-: | :-: | :-: |
+| 1 | ![][flag_JP] Japan | 6 | 1 | 2 |
+| 2 | ![][flag_TW] Taiwan | 2 | 4 |  |
+| 3 | ![][flag_HK] Hong Kong |  | 3 | 1 |
+| 4 | ![][flag_FR] France |  |  | 2 |
+| 5 | ![][flag_CA] Canada |  |  | 2 |
+| 5 | ![][flag_CN] China |  |  | 1 |
+
+#### Attendance
+
+| Country | 2011 | 2012 | \#3 | 2014 | 2015 | 2016 | 2017 | 2018 |
```

Comment:
Would be better to replace (1st) (2nd) and (3rd) with the crowns?

Reply:
Yeah, will apply that.